### PR TITLE
KT-20357: Add samples for any, all and none

### PR DESCRIPTION
**Pull request by yusaka39**
https://youtrack.jetbrains.com/issue/KT-20357

Add samples for `any`, `all` and `none` and regenerate stdlib codes.

### CODE REVIEW

File: libraries/stdlib/samples/test/samples/aggregates/aggregates.kt [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+ */
+
+package samples.aggregates
+
+import samples.Sample
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+
+class Aggregates {
```
Comment (ilya-g) [inline libraries/stdlib/samples/test/samples/aggregates/aggregates.kt]: Better to place it as a nested class of `samples.collections.Collections`

File: libraries/stdlib/samples/test/samples/aggregates/aggregates.kt [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    @Sample
+    fun all() {
+        val isEven: (Int) -> Boolean = { it % 2 == 0 }
+        val zeroToTen = 0..10
+        assertFalse {
+            zeroToTen.all(isEven)
```
Comment (ilya-g) [inline libraries/stdlib/samples/test/samples/aggregates/aggregates.kt]: Though we can definitely use `all`/`any`/`none` with a callable reference, the most common usage is with a lambda function: `.all { isEven(it) }`. Probably, we can show both options.

**Comment by YutaSakata** [comment]
@ilya-g
Thank you for your review. I updated and squashed the branch.

**Comment by yusaka39** [comment]
@ilya-g
Thank you for your review. I updated and squashed the branch.

**Comment by ilya-g** [comment]
Thank you, I've merged this PR manually with minor edits.

**Pull request closed without merging**